.gitignore

```
@@ -0,0 +1,8 @@
+# webstorm
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
```

```suggestion
.idea
```
일일이 파일 ignore 시켜주실 필요없고 .idea만 기입해두시면 일괄처리됩니다.
다음번 PR 때 해당 내용으로 변경해서 포함시켜주시면 감사드리겠습니다

[.gitignore 변경사항 적용](https://brunch.co.kr/@hopeless/9)

---

넵!